docs/codespaces.md

```diff
@@ -19,13 +19,30 @@ You can create and access a codespace either via the browser or by connecting di
 You only need to do the following the first time you set up the codespace.
 
 ### Add Environment Variables
-- Copy the `sample.env` (run commands from within the remote container, i.e. browser or VS Code terminal)
+- Copy the `codespaces.sample.env` (run commands from within the remote container, i.e. browser or VS Code terminal)
 ```bash
-cp sample.env .env
+cp codespaces.sample.env .env
 ```
 - Add `POSTGRES_URL="postgresql://digitalmarketplace:digitalmarketplace@localhost:5432/digitalmarketplace"` to the `.env` file.
 - Add additional desired environment variables to the `.env` file (refer to the [Environment Variables](https://github.com/button-inc/digital_marketplace/blob/main/README.md#environment-variables) section in the README)
```

Comment:
Do you agree? I panicked that it was failing on the browser and then realized it was just that I hadn't set these there, and maybe I can save someone else the frustration
```suggestion
- Add additional desired environment variables to the `.env` file (refer to the [Environment Variables](https://github.com/button-inc/digital_marketplace/blob/main/README.md#environment-variables) section in the README). At minimum, to be able to start the app, you need to add `MAILER_GMAIL_USER` and `MAILER_GMAIL_PASS`.
```